daq/forchestrator.py

```diff
@@ -40,15 +42,27 @@ def _handle_faucet_events(self):
             LOGGER.debug('Faucet event %s', event)
             if not event:
                 return True
+
+            timestamp = event.get("timestamp", time.time())
+
             (dpid, port, active) = self._faucet_events.as_port_state(event)
             if dpid and port:
                 LOGGER.info('Port state %s %s %s', dpid, port, active)
+                port_state_event = faucet_events.PortStateEvent(dpid, timestamp, port, active)
+                faucet_events.process_port_state(port_state_event)
+
             (dpid, port, target_mac) = self._faucet_events.as_port_learn(event)
             if dpid and port:
                 LOGGER.info('Port learn %s %s %s', dpid, port, target_mac)
+                port_learn_event = faucet_events.process_port_learn(dpid, timestamp, port, target_mac)
```

Comment:
nb: timestamp should be before dpid b/c you can have a timestamp w/o a dpid, but you can't have a dpid w/o a timestamp....